source/conf.py

```diff
@@ -214,3 +216,10 @@
 
 # If true, `todo` and `todoList` produce output, else they produce nothing.
 todo_include_todos = True
+
+autodoc_typehints = "none"
```

Comment:
@aroberge this doesn't do anything for now, but when I add type hints in `friendly_traceback`, Sphinx will start to spit warnings since the types can't be resolved, so turned off type hints preliminary.